push-build.sh

```diff
@@ -125,7 +125,7 @@ KUBE_ROOT=$(pwd -P)
 # TODO: this should really just read the version from the release tarball always
 USE_BAZEL=false
 if release::was_built_with_bazel $KUBE_ROOT $FLAGS_release_kind; then
-  USE_BAZEL=true
+  USE_BAZEL=false
```

Comment:
I don't think this actually works because of the paths, but it would be so much easier if it did ;-)